examples/init/INCAR_methane.rlx

```diff
@@ -11,8 +11,8 @@ NELMIN=4
 ISIF=2
 ISMEAR=0
 SIGMA=0.20000
-IBRION=2
-NSW=50
+IBRION=-1
+NSW=0
```

Comment:
Why do you introduce the change here?

Reply:
This INCAR is the original version of our methane tutorials for DP-GEN.  I'm reorganizing the tutorials, so I put the original files.  In methane I chose a very relaxed structure as initial structure, so "IBRION=-1"  is enough, there's no need to move ions. But I admit that "IBRION=2" and "NSW=50" may be general

Reply:
The INCAR file has an extension `.rlx`, which indicates that it is for relaxation, so it is better to set IBRION=2. Otherwise if the INCAR is not for relaxation, we should remove `.rlx`

Reply:
Fixed